plugins/core/webview/src/q-ui/components/awsProfileForm.vue

```diff
@@ -10,16 +10,16 @@
             />
         </svg>
     </button>
-    <div class="title no-bold">Profile Name</div>
-    <div class="hint">The identifier for these credentials</div>
+    <div class="title no-bold font-amazon">Profile Name</div>
+    <div class="hint font-amazon">The identifier for these credentials</div>
```

Comment:
can't we just set the default body font instead of needing to remember to apply the font to every element?

Reply:
yea we should lol, I will fix it later